doc/rippled-example.cfg

```diff
@@ -606,30 +606,49 @@
 #   Partial pathnames will be considered relative to the location of
 #   the rippled.cfg file.
 #
-#   [node_db]       Settings for the NodeDB (required)
-#   [temp_db]       Settings for the look-aside temporary db (optional)
-#   [import_db]     Settings for performing a one-time import (optional)
+#   [node_db]               Settings for the Node Database (required)
 #
 #   Format (without spaces):
-#       One or more lines of key / value pairs:
+#       One or more lines of case-insensitive key / value pairs:
 #       <key> '=' <value>
 #       ...
 #
-#   Examples:
-#       type=rocksdb
-#       path=db/rocksdb
+#   Example:
+#       type=nudb
+#       path=db/nudb
+#
+#   The "type" field must be present and controls the choice of backend:
+#
+#   type = NuDB
+#
+#       NuDB is high performance, but only works with Solid State (SSD)
+#       Drives or installations that have higher IOPS. It maintains its
+#       high speed regardless of the amount of history stored. Online
+#       delete may be selected, but is not required. NuDB is available
+#       on all platforms that rippled runs on.
+#
+#   type = RocksDB
+#
+#       This can work with spinning disks or installations that have
+#       lower IOPS (read/write throughput). It is not suitable for
+#       storing full history because it slows down as the data set
+#       increases. If RocksDB is chosen as the backend, we suggest
+#       also setting up online delete. RocksDB is not available
+#       on Windows environments.
+#
+#       The RocksDB backend also provides these optional parameters:
+#
+#       compression         0 for none, 1 for Snappy compression
+#
 #
-#   Choices for 'type' (not case-sensitive)
-#       RocksDB             Use Facebook's RocksDB database (preferred)
-#       NuDB                Use Ripple Labs' NuDB (Windows preferred)
-#       SQLite              (Deprecated)
-#       none                (No backend)
 #
 #   Required keys:
 #       path                Location to store the database (all types)
 #
 #   Optional keys:
-#       compression         0 for none, 1 for Snappy compression
+#
+#       These keys may be configured for any type of backend:
```

Comment:
This puzzled me for a bit.  But the keys themselves aren't "configured".  What you mean is that the following keys are [valid, legal, acceptable] for any backend, yes?